### DISCUSSION TITLE
+layout from package

### DISCUSSION THREAD

**Discussion by bartcuijpers** [sveltejs/kit#12246] [Q&A]
Hi,

I was wondering if it's possible to provide a +layout file from an external package.
Or should I create an 'installer' script that copies/overwrites certain files in the src tree of the parent project?

The use case I'm thinking of is building an admin dashboard skeleton that can be updated like a regular npm package.

Any help is welcome :)

**Comment by david-plugge** [comment]
No, thats ist not possible in the way you described. However, you can build a component/function that can be used in the +layout file instead. This also results in a more customizable App for the Users of your package. In Addition to that you could provide a cli to scaffold the project with your package
